ui/src/features/components/Report/index.js

```diff
@@ -45,6 +47,18 @@ class Report extends React.Component {
         this.setState({disabledCreateBenchmark: true})
     };
 
+    exportCSV = () => {
+        const {report} = this.props;
+        axios.get(`${process.env.PREDATOR_URL}/tests/${report.test_id}/reports/${report.report_id}/export/csv`)
```

Comment:
also here, use window.open instead of axios. The browser will take care of the download file.

Reply:
I agree, we need the API to return with download

https://stackoverflow.com/questions/7288814/download-a-file-from-nodejs-server-using-express

Reply:
Thank you for the suggestion. I have switched to using `window.open()` for the downloads.